docs/reference/filters.md

```diff
@@ -2437,3 +2437,27 @@ pr: Path("/products/:productId")
 consistentHashKey("${request.header.Authorization}")
 consistentHashKey("${request.source}") // same as the default key
 ```
+
+## consistentHashBalanceFactor
+
+This filter sets the balance factor used by the [`consistentHash`](backends.md#load-balancer-backend) algorithm to prevent a single backend endpoint from being overloaded.
+The number of in-flight requests for an endpoint can be no higher than `(average-in-flight-requests * balanceFactor) + 1`.
```

Comment:
I am not sure where does `+ 1` comes from, the impl uses 
```go
targetLoad := averageLoad * balanceFactor
...
if load <= int(targetLoad) {
    break
}
``` 
Lets remove and simplfy?

Reply:
After we assign a request to the server the number of in-flight requests will be targetLoad + 1